yang-schema-comparison/draft-ietf-netmod-yang-schema-comparision.xml

```diff
@@ -33,6 +33,122 @@
     </abstract>
   </front>
   <middle>
+    <section anchor="key-issues" numbered="true" toc="default">
+      <name>Key Issues</name>
+      <t>{ This section is only to present the current ongoing work, not part of
+      the final draft. }</t>
+      <t>The contributors have identified several key issues that need
+      attention. This section presents selected key issues which have been
+      discussed together with suggestions for proposed solution or
+      requirements.</t>
+      <t>So far, the following issues have been discussed and the following
+          reflections or suggested solutions have been reached.</t>
+      <section>
+        <name>On-wire vs Schema analysis</name>
+        <t>Should one algorithm be used or two? The consesus reached was to
```

Comment:
It would be good to provide an example of what these two cases mean.  If not in the draft, we need them for the slides.  But having that here will help people dive right into this issue.

Reply:
Is it enough to mention that for on-wire algorithm a type is not necessarily NBC, but it would be for the schema algorithm?

Reply:
Perhaps.  What I was getting at here is more description of the on-wire vs. schema problem to set context.  I can't remember the example on the previous call, but I recall it was a good one.

Reply:
Per - instead of "a type change" perhaps better to say "a typedef name change, or a choice name change"

Reply:
On the wire: the focus is on what types of changes affect the NETCONF/RESTCONF/gNMI/etc requests and responses from/to a client. If the same requests and responses occur, then there is no "on the wire" impact of the change. For example - changing the name of a 'choice' has no impact "on the wire". For many clients, this level of compatibility is enough.

Schema: any changes that affect the YANG schema in an NBC manner according to the full rules of Module Versioning. This may be important for clients that, for example, auto-code-gen using the YANG and where the change of a typedef name or a choice name could be significant. Also important for other modules that may augment or deviate the schema being compared.

Reply:
Added your text with slight rewording @jsterne